packages/web-worker/src/webpack-parts/loader.ts

```diff
@@ -0,0 +1,124 @@
+import * as path from 'path';
+
+import SingleEntryPlugin from 'webpack/lib/SingleEntryPlugin';
+import WebWorkerTemplatePlugin from 'webpack/lib/webworker/WebWorkerTemplatePlugin';
+import FetchCompileWasmTemplatePlugin from 'webpack/lib/web/FetchCompileWasmTemplatePlugin';
+
+import {WebWorkerPlugin} from './plugin';
+
+const NAME = 'WebWorker';
+
+export function pitch(
+  this: import('webpack').loader.LoaderContext,
+  request: string,
+) {
+  this.cacheable(false);
+  const callback = this.async();
+
+  const {
+    context,
+    resourcePath,
+    _compiler: compiler,
+    _compilation: compilation,
+  } = this;
+
+  // if (compiler.options.output.globalObject !== 'self') {
```

Comment:
🔥 ?

Reply:
I left this in as a reminder to myself that I still have to figure out the right approach for these scripts on the server render :P